lib/view/disclaimer_view.dart

```diff
@@ -122,62 +122,65 @@ class _DisclaimerViewState extends State<DisclaimerView> {
                           crossAxisAlignment: CrossAxisAlignment.start,
                           children: <Widget>[
                             Container(
-                              width: 42.0,
+                              margin: const EdgeInsets.only(right: 14.0),
                               child: const Text(
                                 '✋',
                                 style: Styles.textH3,
                               ),
                             ),
-                            const Expanded(
-                              child: Text(
-                                'Please keep in mind\n',
-                                style: Styles.textH3,
-                              ),
-                            ),
+                            Expanded(
+                                child: Column(
+                              crossAxisAlignment: CrossAxisAlignment.start,
+                              children: const <Widget>[
+                                Text(
+                                  'Please keep in mind',
+                                  style: Styles.textH3,
+                                ),
+                                SizedBox(height: 10.0),
+                                Text(
+                                  'This is not a comprehensive source nor can we guarantee it is completely up to date at '
+                                  'the time of use 📱.\n\nIt is created using Western Health guidelines, informally '
+                                  'peer-reviewed and adapted, with permission, from College/Society guidelines.\n\n'
+                                  'WHAC19 gathers analytics and crash data which is used to improve the app.  For more information and to opt out, go to Information > Privacy\n\n',
+                                  style: Styles.textBody,
+                                )
+                              ],
+                            ))
```

Comment:
Trailing comma here I think should also make sure dartfmt does it right so ),]

Reply:
Oh I know. It needs comma everywhere.